tests/utils/nitpick.js

```diff
@@ -505,3 +561,55 @@ module.exports.saveResults = function(passed, resultsObject) {
 		++textIndex;
     }
 }
+
+module.exports.verifyClientProfile = function() {
+    for (var i = 0; i < arguments.length; ++i) {
+        if (arguments[i] == "Any") {
+            console.warn("Running on 'Any platform'");
+            return true;
+        } else if (arguments[i] == "VR-High") {
```

Comment:
I think we might want more details here?  Something like <Headset/Desktop>-<Windows/Mac/Android>-<High/Low> (with the last one being optional).

So you'd have things like "Rift-Windows-High", "Desktop-Mac-Low", "Quest-Android".  Or something along those lines, to make this system more extendable in the future.

Reply:
I haven't implemented Quest yet as I have no idea how to detect that we are running on Quest.  I also don't know how we will run tests on mobile devices.  I don't think we want to write 100's of images to memory. Can we run the test scripts on a mobile device - they need command line parameters...